rad/prelude/error-messages.rad

```diff
@@ -82,11 +82,11 @@
 
 (def uppercase-item
   (fn [item]
-    (lookup item {:issue "Issue" :patch  "Patch"})))
+    (lookup item {:issue "Issue" :patch "Patch" :project "Project"})))
 
 (def lowercase-item
   (fn [item]
-    (lookup item {:issue "issue" :patch  "patch"})))
+    (lookup item {:issue "issue" :patch "patch" :project "project"})))
```

Comment:
```
(def lowercase-item [item]
  (drop 1 item))
```

Reply:
The wording changed. In the code `registered-machine` is used. In the UI we decided on `Radicle machine` so the flags and item names don't always match.